cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -148,4 +148,8 @@ enable_use_persist_log_on_disk_to_catch_up=true
 
 # The number of logs read on the disk at one time, which is mainly used to control the memory usage.
 # This value multiplied by the log size is about the amount of memory used to read logs from the disk at one time.
-max_number_of_logs_per_fetch_on_disk=1000
\ No newline at end of file
+max_number_of_logs_per_fetch_on_disk=1000
+
+# When consistency level set to mid, query failed if the log lag exceeds max_read_log_lag
+# This default value is 1000
+max_read_log_lag=1000
```

Comment:
Perhaps we need to describe in detail what this parameter represents, such as with the `insertRecord()` interface, where a single log represents a row data, whereas if the `insertTablet()` interface is used, where a single log may represent several thousand rows of data (depending on the size of the tablet set by the user), these things need to be known to users to set this parameter reasonably according to their business logic.